warehouse/templates/manage/account.html

```diff
@@ -504,6 +504,20 @@ <h2>{% trans %}API tokens{% endtrans %}</h2>
   <section id="account-events">
     <h2>{% trans %}Security history{% endtrans %}</h2>
 
+    {% set role_strings = {
```

Comment:
I think this method is clean but it deviates from existing code, LMK if you don't like it.